elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "PaackEng/paack-ui",
     "summary": "Paack's Design System applied over Elm UI",
     "license": "BSD-3-Clause",
-    "version": "2.1.0",
+    "version": "3.0.0",
```

Comment:
Ok, with the information you gave me in Slack I see the whole situation.
We won't need to bump to 3.0.0 if we do `NavigationContainer.withMenuStyle", supposing that all pages will use the same sidebar style...